connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaOffsetBackingStore.java

```diff
@@ -59,21 +61,21 @@
     private HashMap<ByteBuffer, ByteBuffer> data;
 
     @Override
-    public void configure(Map<String, ?> configs) {
-        String topic = (String) configs.get(OFFSET_STORAGE_TOPIC_CONFIG);
+    public void configure(WorkerConfig config) {
+        String topic = config.getString(DistributedConfig.OFFSET_STORAGE_TOPIC_CONFIG);
         if (topic == null)
```

Comment:
Same question about `null` check here too -- I think this is now required, so would have already thrown a config execption?